public/editor/scripts/project-list.js

```diff
@@ -78,6 +78,13 @@ require(["jquery", "constants", "analytics", "moment"], function($, Constants, a
 
   $("#project-list").prepend(favoriteProjectsElementList);
 
+  $(".remix-link").click(function() {
```

Comment:
@kramsamujh Instead of adding it via a ``click``handler. Could you please set the URL as an ``href`` attribute to the ``a.remix-links``. You can do that in the ``forEach`` call on line ``67``. I think this is better since then you can open the link in a new window if you want, and you can also see the URL to the remix when you hover your mouse over the link.

Thanks!